root/components/list/ReleaseList.js

```diff
@@ -163,24 +169,24 @@ const ReleaseList = ({
       });
 
       return [
-        ...(checkboxColumn ? [checkboxColumn] : []),
-        ...(seriesNumberColumn ? [seriesNumberColumn] : []),
+        checkboxColumn,
+        seriesNumberColumn,
         nameColumn,
         artistCreditColumn,
         formatColumn,
         tracksColumn,
         releaseEventsColumn,
-        ...(labelsColumn ? [labelsColumn] : []),
+        labelsColumn,
         catnosColumn,
         barcodeColumn,
-        ...(showLanguages ? [releaseLanguageColumn] : []),
-        ...(instrumentUsageColumn ? [instrumentUsageColumn] : []),
-        ...(typeColumn ? [typeColumn] : []),
-        ...(statusColumn ? [statusColumn] : []),
-        ...($c.session?.tport == null ? [] : [taggerColumn]),
-        ...(showRatings ? [ratingsColumn] : []),
-        ...(mergeForm && releases.length > 2 ? [removeFromMergeColumn] : []),
-      ];
+        releaseLanguageColumn,
+        instrumentUsageColumn,
+        typeColumn,
+        statusColumn,
+        ($c.session?.tport == null) ? null : taggerColumn,
+        showRatings ? ratingsColumn : null,
+        (mergeForm && releases.length > 2) ? removeFromMergeColumn : null,
+      ].filter(Boolean);
```

Comment:
It wasn't strictly required, but was semi-required, because Flow now wants us to annotate the empty array literals `[]`, which would've made things a lot uglier here. Changing those to null and filtering them out ended up being a lot cleaner.